tests/src/DeprecationRulesTest.php

```diff
@@ -19,19 +19,27 @@ public function testDeprecationRules(string $path, int $count, array $errorMessa
 
     public function dataDeprecatedSamples(): \Generator
     {
+//        yield [
+//            __DIR__ . '/../fixtures/drupal/modules/phpstan_fixtures/src/UsesDeprecatedUrlFunction.php',
+//            2,
+//            [
+//                '\Drupal calls should be avoided in classes, use dependency injection instead',
+//                'Call to deprecated method url() of class Drupal.'
+//            ]
+//        ];
+//        yield [
+//            __DIR__ . '/../fixtures/drupal/core/lib/Drupal/Core/Entity/EntityManager.php',
+//            1,
+//            [
+//                'Class Drupal\Core\Entity\EntityManager implements deprecated interface Drupal\Core\Entity\EntityManagerInterface.'
+//            ]
+//        ];
         yield [
-            __DIR__ . '/../fixtures/drupal/modules/phpstan_fixtures/src/UsesDeprecatedUrlFunction.php',
+            __DIR__ . '/../fixtures/drupal/modules/phpstan_fixtures/src/DeprecatedGlobalConstants.php',
             2,
             [
-                '\Drupal calls should be avoided in classes, use dependency injection instead',
-                'Call to deprecated method url() of class Drupal.'
-            ]
-        ];
-        yield [
-            __DIR__ . '/../fixtures/drupal/core/lib/Drupal/Core/Entity/EntityManager.php',
-            1,
-            [
-                'Class Drupal\Core\Entity\EntityManager implements deprecated interface Drupal\Core\Entity\EntityManagerInterface.'
+                'Call to deprecated constant DATETIME_STORAGE_TIMEZONE.',
+                'Call to deprecated constant DATETIME_DATE_STORAGE_FORMAT.',
```

Comment:
These should be failing now as we should now get the deprecation message as well.